src/core/verification.service.ts

```
@@ -150,12 +308,36 @@
+            if (match && match[1]) {
+              keyId = match[1];
+            }
+          } catch (e: unknown) {
+            logger.warn(
+              `Error parsing verificationMethod URL: ${proofInput.verificationMethod}`,
+              { message: (e as Error).message }
+            );
+          }
+        }
+
+        return await this.verifyJWTProofForAssertion(proofInput, keyId);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)